app/styles/app.sass

```diff
@@ -3312,3 +3316,19 @@ $filetag-bg: #f3f5f8
   &-item
     height: 1.6em
     border-radius: 0.2em
+
+.flex-2
+  flex: 2
+.flex-3
+  flex: 3
+.warning-msg
+  height: 30px;
+
+.margin-bottom-5
+  font-weight: 900;
+  font-size: 18px;
+  margin-bottom: 5px;
```

Comment:
For bottom margins use already existing `margin-b-*` classes. 

Also, util class with name `margin-bottom-5` should never contain other styles like font style modifiers, use specific class for the target element or use existing util classes like `font-medium`, `bold` etc.